go/worker/compute/executor/committee/node.go

```diff
@@ -150,6 +149,9 @@ type Node struct { // nolint: maligned
 	// safely be used without holding the lock.
 	schedulerMutex sync.RWMutex
 	scheduler      schedulingAPI.Scheduler
+	// Per round weight limits that should be taken into account when scheduling
+	// transactions.
+	roundWeightLimits map[string]uint64
```

Comment:
Why is this member variable needed? It seems to only be used during parameter update? It would be safer if this was removed.

Reply:
Yeah indeed not needed at this time (until there is support for custom runtime per-round weight limits). Fixed